src/app/components/list/toolbar/index.html

```diff
@@ -128,13 +128,44 @@ <h3>
   </sky-demo-page-example>
 	
 	<sky-demo-page-content sectionHeading="UX guidelines">
-		<div class="sky-subsection-heading">Sorting</div>
-		<p>If you use the toolbar with a grid view, you have two options for letting users sort the list:</p>
-		<ul>
-			<li>The sort button in the toolbar</li>
-			<li>Clickable column headers</li>
-		</ul>
-		<p>Don't combine both sorting options in a single list because it confuses users about which option is currently applied.</p>
+		
+		<section>
+			<header><div class="sky-section-heading">Usage</div></header>
+			<stache-row>
+				<stache-column screenSmall="6">
+					<p>
+						The list toolbar appears above a <a routerLink="/components/list">list</a> of records. It contains actions to manipulate the contents and display of the list. Place frequently used actions directly in the toolbar, and place less-frequently used actions in a more actions menu. When multiple views of the list are available, the view switcher is docked on the right side of the toolbar.
+					</p>
+				</stache-column>
+				<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info"><img src="~/assets/img/guidelines/toolbar/list-toolbar-context.png" style="width: 100%;" /></div>
+				</stache-column>
+			</stache-row>
+		</section>
+		
+		<section>
+			<header><div class="sky-section-heading">Composition</div></header>
+			<div class="screenshot screenshot-info"><img src="~/assets/img/guidelines/toolbar/anatomy.png" style="width: 100%;" /></div>
+			<p>For more information about how to arrange the content in a list toolbar see the <a routerLink="/components/toolbar">toolbar</a> UX guidelines.</p>
```

Comment:
I'd delete "the" before "content." ... Add a comma after "toolbar" and before "see." ... And I'd update the link to link directly to the UX guidelines section, changing it from a `routerLink` to a `stacheRouterLink` and adding a `fragment`: `stacheRouterLink="/components/toolbar" fragment="ux-guidelines"`. I'd also make "UX guidelines" part of the link text.

Reply:
Ok. What does stacheRouterLink do that routerLink doesn't?

Reply:
I think `stacheRouterLink` is largely the same as `routerLink` but that it includes the `fragment` property to allow links to anchor points. If you know how to make links to the "UX guidelines" anchor point work with `routerLink` instead, that works just as well. I just suggested switching to `stacheRouterLink` because I knew I would be able to get the link to the anchor tag to work with the `fragment` attribute. And I tested it locally to confirm that it would work.